File: WebContent/js/service/horarioNombreService.js
'use strict';
app.service('horarioNombreService',
	    function horarioNombreService($resource) {
	        return $resource('services/horarioNombreServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario/:nbHorarioModif', {
	        	nbModulo: '@nbModulo',nbGrupoModulo: '@nbGrupoModulo',nbSubGrupoModulo: '@nbSubGrupoModulo',nb_actividad:'@nb_actividad',nbHorario:'@nbHorario',nbHorarioModif:'@nbHorarioModif'
	        }, {
	            query: {
	            	url:'services/horarioNombreServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad',
	                method: 'GET',
	                isArray: true
	            },
	            
	            create: {
	            	url:'services/horarioNombreServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario',
	                method: 'POST',
	                isArray: false
	            }
	            ,
	            queryNoExiste: {
	            	url:'services/horarioNombreServiceResource/queryNoExiste/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario',
	                method: 'GET',
	                isArray: false
	            }
	            ,
	            isDelete: {
	            	url:'services/horarioNombreServiceResource/isDdelete/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario',
	                method: 'get',
	                isArray: false
	            },
	            deletex: {
	            	url:'services/horarioNombreServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario',
	                method: 'DELETE',
	                isArray: false
	            } ,
	            update: {
	            	url:'services/horarioNombreServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nb_actividad/:nbHorario/:nbHorarioModif',
	                method: 'PUT',
	                isArray: false
	            }
	        });
	    });

 

File: src/ve/org/bcv/services/HorarioNombreService.java
package ve.org.bcv.services;

import java.util.List;

import ve.org.bcv.dto.DeleteYesDto;
import ve.org.bcv.dto.HorarioNombreDto;

public interface HorarioNombreService {
	List<HorarioNombreDto> findByActividadesByModulo(String nbModulo, String nbGrupoModulo,String nbSubGrupoModulo,String actividad);
	HorarioNombreDto delete(String nbModulo, String nbGrupoModulo, String nbSubGrupoModulo, String nbActividad, String nbHorario)throws Exception;
	HorarioNombreDto save(String nbModulo, String nbGrupoModulo, String nbSubGrupoModulo,String actividad, String nbHorario)throws Exception;
	HorarioNombreDto update(String nbModulo, String nbGrupoModulo, String nbSubGrupoModulo, String nbActividad,
			String nbHorario, String nbHorarioModif) throws Exception;
	Boolean noExiste(String nbModulo, String nbGrupoModulo, String nbSubGrupoModulo,String actividad,String nbHorario)
			throws Exception;
	 DeleteYesDto isDelete(String nbModulo, String nbGrupo, String nbSubGrupoModulo, String actividad, String nbHorario) ;
}
